stats.js

```diff
@@ -66,18 +67,7 @@ const getTestingData = async () => {
   } catch (err) {
     maxAge = defaultMaxAge
   }
-  let data = await getStateWideTestingData(maxAge)
-  data = data
-    .map(record => {
-      if (
-        Date.parse(record.test_date) + maxAge <
-        new Date().getTime() - 1000 * 60 * 60 * 24
-      ) {
-        return false
-      }
-      return record
-    })
-    .filter(item => !!item)
+  let data = await getStateWideTestingData()
```

Comment:
Regarding the todos you posted, if you want to avoid processing the entire dataset, assuming this function gets you data from within the provided `maxAge`, you could pass it `maxAge + 1000 * 60 * 60 * 24 * movingAvgDays` which should get you only the data you need to calculate the averages and totals

Reply:
Scratch that, just saw this function defaults to `10000` on the first argument 🤔